views/forms/referral_form.php

```diff
@@ -153,7 +153,7 @@
     $form_type = "agency referral";
 
     // First card (Participant Information)
-    $pers_firstname = !empty($_POST['pers_firstname']) ? strtolower((trim($_POST['pers_firstname']))) : NULL;
+   $pers_firstname = !empty($_POST['pers_firstname']) ? strtolower((trim($_POST['pers_firstname']))) : NULL;
```

Comment:
Do you still need to the `strtolower()` if the query is now searching based on lowercase?